roles/prereqs/libselinuxpython/tasks/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 
 - name: install libselinux python
-  yum:
+  package:
```

Comment:
I need to review the way of manage packages I think I used yum instead of package